addons/hr/static/src/core/web/thread_actions.js

```diff
@@ -35,12 +34,9 @@ threadActionsRegistry.add("open-hr-profile", {
                 ["id"]
             );
             employeeId = employees[0]?.id;
-        }
-        if (employeeId) {
-            store.Persona.insert({
-                ...component.thread.correspondent,
-                employeeId,
-            });
+            if (employeeId) {
+                component.thread.chatPartner.employeeId = employeeId;
```

Comment:
This could/should/might be renamed `chatPersona` in an other PR https://github.com/odoo/odoo/pull/139078/files#diff-7201775967110993e33c6484705a4d8ed9ff25ecb2ba7bf3a08af409f68bfaa3